app/views/sign_up/registrations/show.html.slim

```diff
@@ -40,8 +41,5 @@
           p.mb2 = t('devise.registrations.start.bullet_5_html')
           p.mb3 = link_to \
             t('devise.registrations.start.learn_more'), MarketingSite.help_url, target: '_blank'
-  .center
-    - ab_test(:demo) do |variant, _|
```

Comment:
This is not removing the A/B testing entirely. It is removing a duplicate div that was causing the LOA3 spec to fail because it found 2 elements matching the "Get started" button. I suppose I could edit the spec to use the first match and then remove this in a separate PR. What do you think?